tests/matrix_free/fe_remote_evaluation_01.with_p4est=true.mpirun=1.output

```diff
@@ -0,0 +1,21 @@
+
+DEAL:0::Setup MatrixFree...
+DEAL:0::
+Test the access of remote values at points in face batches...
+DEAL:0::  Setup Communicator...
+DEAL:0::  Test with first_selected_component = 0
+DEAL:0::    accumulated error: 9.12638e-15 8.11851e-15
+DEAL:0::  Test with first_selected_component = 1
+DEAL:0::    accumulated error: 9.12638e-15 8.11851e-15
+DEAL:0::  Test with first_selected_component = 2
+DEAL:0::    accumulated error: 9.12638e-15 8.11851e-15
```

Comment:
You don't need to output these values. Simply assert that the errors are smaller than `X`.